test/integration/quic_http_integration_test.cc

```diff
@@ -417,5 +417,37 @@ TEST_P(QuicHttpIntegrationTest, ResetRequestWithoutAuthorityHeader) {
   EXPECT_EQ("400", response->headers().getStatusValue());
 }
 
+TEST_P(QuicHttpIntegrationTest, ResetLargeResponseUponReceivingHeaders) {
```

Comment:
Can you please move this to protocol_integration_test so it also runs for HTTP1 and HTTP2 like I intended when I originally crafted this test case?